app/assets/javascripts/update_inline.js

```diff
@@ -0,0 +1,46 @@
+function updateInline() {
+  $('.ideas').delegate('.title', 'click', function() {
+    var $idea = $(this).closest(".idea");
+    document.getElementById("ideaTitle" + $idea.attr('data-id')).contentEditable = true;
+    $("#ideaTitle" + $idea.attr('data-id')).keyup(function(event){
```

Comment:
You're already grabbing this up above, you might as well store it in a variable, right?